src/components/Table/GridTable.tsx

```diff
@@ -308,7 +319,7 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
     const filteredRows: RowTuple<R>[] = [];
 
     // Misc state to track our nested card-ification, i.e. interleaved actual rows + chrome rows
-    const nestedCards = !!style.nestedCards && new NestedCards(columns, filteredRows, style);
+    const nestedCards = !!style.nestedCards && new NestedCards(columns, style);
```

Comment:
Removed the ability to instantiate the NestedCards class with a buffer to add the Chrome rows to. Instead we are making it explicit on which buffer to add each Chrome row.

Reply:
Cool, makes sense

Reply:
...an alternative would be to make two `NestedCards`, i.e.

```
const rowCards  = !!style.nestedCards && new NestedCards(columns, filteredRows, style);
const headerCards = !!style.nestedCards && !!style.nestedCards["header"] && new NestedCards(columns, headerRows, style);
```

Then in the `if row.kind === header` case, use `headerCards` instead of `nestedCards`.

Reply:
You got it!